lumigator/python/mzai/backend/backend/models.yaml

```diff
@@ -0,0 +1,95 @@
+- name: facebook/bart-large-cnn
+  uri: hf://facebook/bart-large-cnn
+  description: BART is a large-sized model fine-tuned on the CNN Daily Mail dataset.
+  info:
+    parameters_count: 406M
+    tensor_type: F32
+    model_size: 1.63GB
+  tasks:
+    - summarization:
+        max_length: 142
+        min_length: 56
+        length_penalty: 2.0
+        early_stopping: true
+        no_repeat_ngram_size: 3
+        num_beams: 4
+
+- name: mikeadimech/longformer-qmsum-meeting-summarization
+  uri: hf://mikeadimech/longformer-qmsum-meeting-summarization
+  description: Longformer is a transformer model that is capable of processing long sequences.
+  info:
```

Comment:
[Not strictly necessary] I imagine this info is included here for convenience to later be shown on the UI? Perhaps it would (instead or also) be helpful to provide the link to the [model card](https://huggingface.co/facebook/bart-large-cnn).

Reply:
I didn't do that because not every model card has something useful, so, I would suggest to leave it out.

Reply:
Fair point, I still think we should provide the link, just the link, but it doesn't have to be part of this PR.